app/models/micropost.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Validates Micropost
+class Micropost < ApplicationRecord
+  belongs_to :user
+  validates :content, length: { maximum: 140 },
```

Comment:
Create a file called lib/constants.rb and extract 140 to a constant. This is how we structure all our apps. See IO and DM for example and do it the same.